content/docs/1.6.0/references/storage-class-parameters.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Storage Class Parameters
+weight: 1
+---
+
+## Overview
+
+Storage Class as a resource object has a number of settable parameters.  Here's a sample YAML:
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: longhorn-test
+provisioner: driver.longhorn.io
+allowVolumeExpansion: true
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+parameters:
+  numberOfReplicas: "3"
+  staleReplicaTimeout: "2880"
+  fromBackup: ""
+  fsType: "ext4"
+  diskSelector: "ssd,fast"
+  nodeSelector: "storage,fast"
+#  recurringJobSelector: '[{"name":"snap-group", "isGroup":true},
+#                          {"name":"backup", "isGroup":false}]'
+```
+
+Some fields are common to all Kubernetes storage classes.
+See also [Kubernetes Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes).  
+
+#### Provisioner
+Specifies the plugin that will be used for dynamic creation of persistent volumes.  For Longhorn, that is always "driver.longhorn.io"
+
+#### Allow Volume Expansion
+> Default: `true`
+
+#### Reclaim Policy
+> Default: `Delete`
+
+#### Volume Binding Mode
+> Default `Immediate`
+
+## Longhorn-specific Parameters
+Note that some of these parameters also exist and may be specified in global settings.  When a volume is provisioned with Kubernetes against a particular StorageClass, StorageClass parameters override the global settings.
```

Comment:
After the recent discussion about the StorageClass parameters, I prefer to add the below note:
```
These fields will be applied for volume creation only. Neither Longhorn nor Kubernetes is responsible for keeping the current SC parameters with that of the volumes previously created by this SC in sync.
```

Reply:
This is a good addition.